app/views/courses/TeacherClassAssessmentsTable.vue

```diff
@@ -1,5 +1,5 @@
 <template lang="jade">
-  #teacher-class-assessments-table
+  #teacher-class-assessments-table(v-if="course && courseInstance && courseInstance.members.length > 0")
```

Comment:
Why are we checking `course` in this `v-if`?  It looks like only `courseInstance` is accessed later.

Reply:
I agree. I removed the unnecessary if condition and pushed my changes. Thanks!